src/App.css

```diff
@@ -32,6 +32,17 @@ body,
 	max-width: 400px;
 }
 
+.descope-saml-container {
+	border-radius: 15px;
+	box-shadow: 0px 1px 50px 0px #b2b2b2;
```

Comment:
those two css properties are shared with descope container, no need to repeat them

let's add another class `descope-base-container` that will contain both of them, and have in classname add it anyway

```
className={`descope-base-container ${samlConfigContainer}`}
```